test/powershell/Modules/Microsoft.PowerShell.Utility/Write-Stream.Tests.ps1

```diff
@@ -84,23 +84,24 @@ Describe "Stream writer tests" -Tags "CI" {
         }
 
        It "Write-Information outputs an information object" -Pending:($IsOSX) {
-           # redirect the streams is sufficient
-           $result = Write-Information "Test Message" *>&1
-           $result.NativeThreadId | Should Not Be 0
-           $result.ProcessId | Should Be $pid
-           $result | Should BeOfType System.Management.Automation.InformationRecord
-           if ($IsWindows)
-           {
-               # Use Match instead of Be so we can avoid dealing with a potential domain name
-               $result.Computer | Should Match ".*${env:COMPUTERNAME}"
-               $result.User | Should Match ".*${env:USERNAME}"
-           }
-           else
-           {
-               $result.Computer | Should Be $(uname -n)
-               $result.User | Should Be $(whoami)
-           }
-           "$result" | Should be "Test Message"
+            # redirect the streams is sufficient
+            $result = Write-Information "Test Message" *>&1
+            $result.NativeThreadId | Should Not Be 0
+            $result.ProcessId | Should Be $pid
+            $result | Should BeOfType System.Management.Automation.InformationRecord
+            
+            # Use Match instead of Be so we can avoid dealing with a potential domain name
+            $result.Computer | Should Match "^[environment]::MachineName"
```

Comment:
`[environment]::machinename` is a literal here.  However, it's still seems wrong to me.
This would match if $result.computer = "foobar" and [environment]::machinename is "foo".

I think what you want is something like:

```powershell
PS /home/steve> [environment]::MachineName                                                              
slee-ubuntu2
PS /home/steve> $machinename = [environment]::MachineName                                               
PS /home/steve> $regex = "^$machinename$|^$machinename\..*"                                             
PS /home/steve> "slee" -match $regex                                                                    
False
PS /home/steve> "slee-ubuntu2.msft.com" -match $regex                                                   
True
PS /home/steve> "slee-ubuntu2b" -match $regex                                                           
False
```